Makefile

```diff
@@ -0,0 +1,10 @@
+target: run_ngrok run_client
```

Comment:
I know it's a choice that a few npm project takes but why do we need `Makefile`?  

What does npm scripts (https://docs.npmjs.com/cli/v8/using-npm/scripts) does not support?

Reply:
Ok, got it, this is what you use for runningt two task in one process.